release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -242,6 +242,182 @@ To support this feature, OLM now enforces CRD upgrades are safer by ensuring
+* Any CRDs owned by the Operator
+--
++
+[NOTE]
+====
+In a future release, these objects will be automatically labeled for any
+Operators where the CSV was installed using a Subscription.
+====
++
+For example:
++
+----
+$ oc label sub etcd operators.coreos.com/etcd-test="" -n test-project
+$ oc label ip install-6c5mr operators.coreos.com/etcd-test="" -n test-project
+$ oc label csv etcdoperator.v0.9.4 operators.coreos.com/etcd-test="" -n test-project
+$ oc label crd etcdcluster.etcd.database.coreos.com operators.coreos.com/etcd-test="" -n test-project
+$ oc label crd etcdbackups.etcd.database.coreos.com operators.coreos.com/etcd-test="" -n test-project
+$ oc label crd etcdrestores.etcd.database.coreos.com operators.coreos.com/etcd-test="" -n test-project
+----
+
+. Verify your Operator has opted in to the new API.
+
+.. List all `operators` resources:
++
+----
+$ oc get operators --all-namespaces
+
+NAME        AGE
+etcd-test   17m
+----
+
+.. Inspect your Operator's details and note that the objects you labeled are
+represented:
++
+----
+$ oc describe etcd-test -n test-project
+
+Name:         etcd-test
+Namespace:
+Labels:       <none>
+Annotations:  <none>
+API Version:  operators.coreos.com/v2alpha1
+Kind:         Operator
+Metadata:
+  Creation Timestamp:  2020-06-30T16:02:59Z
+  Generation:          1
+  Resource Version:    31411
+  Self Link:           /apis/operators.coreos.com/v2alpha1/operators/etcd-test
+  UID:                 85784d0f-7a2f-407a-96d3-938296d8f8b2
+Status:
+  Components:
+    Label Selector:
+      Match Expressions:
+        Key:       operators.coreos.com/etcd-test
+        Operator:  Exists
+    Refs:
+...
+      Kind:                    CustomResourceDefinition
+      Name:                    etcdclusters.etcd.database.coreos.com
+      API Version:             operators.coreos.com/v1alpha1
+      Conditions:
+        Last Transition Time:  2020-06-30T16:06:45Z
+        Message:               all available catalogsources are healthy
+        Reason:                AllCatalogSourcesHealthy
+        Status:                False
+        Type:                  CatalogSourcesUnhealthy
+      Kind:                    Subscription
+      Name:                    etcd
+      Namespace:               test-project
+      API Version:             operators.coreos.com/v1alpha1
+      Conditions:
+        Last Transition Time:  2020-06-30T16:06:48Z
+        Last Update Time:      2020-06-30T16:06:48Z
+        Status:                True
+        Type:                  Installed
+      Kind:                    InstallPlan
+      Name:                    install-spwmk
+      Namespace:               test-project
+      API Version:             operators.coreos.com/v1alpha1
+      Conditions:
+...
+        Message:               install strategy completed with no errors
+        Reason:                InstallSucceeded
+        Status:
+        Type:
+      Kind:                    ClusterServiceVersion
```

Comment:
there is minor issue https://bugzilla.redhat.com/show_bug.cgi?id=1829735 for displaying for csv

Reply:
we can probably dance around that for the example though.